releng/org.eclipse.nebula.examples.incubation.feature/feature.xml

```diff
@@ -35,14 +35,6 @@
          id="org.eclipse.nebula.widgets.datechooser.example.source"
          version="0.0.0"/>
 
-   <plugin
```

Comment:
If it's removed here.  I expect it would be added somewhere else, i.e., to org.eclipse.nebula.examples.release.feature

Reply:
ack, fixed in commit #2